coalib/collecting/Collectors.py

```diff
@@ -150,11 +150,11 @@ def collect_bears(bear_dirs, bear_globs, kinds, log_printer,
     Collect all bears from bear directories that have a matching kind
     matching the given globs.
 
-    :param bear_dirs:           Directory name or list of such that can contain
+    :param bear_dirs:           directory name or list of such that can contain
```

Comment:
first of all, why are we uncapitalizing them? I think they should stay capitalized, why would we change them?